assets/images.css

```diff
@@ -76,6 +76,8 @@
   align-items: center;
   justify-content: center;
   position: relative;
+  max-width: calc(var(--max-width) + 8px);
```

Comment:
I believe this is causing the background to be wider than the max-width (see screenshot).
Should this be `calc(var(--max-width) - 32px)` to match the 16px padding in other sections?

![Screenshot 2023-11-13 at 11 45 29](https://github.com/booqable/tough-theme/assets/690113/fa42cbf7-bc72-4406-a30d-255ee4cdd12d)

Reply:
Fixed